articles/virtual-machines/workloads/redhat/redhat-rhui.md

```diff
@@ -211,13 +212,14 @@ australiaeast - 20.248.180.252
 southeastasia - 20.24.186.80
 
 # Azure US Government.
-# To be deprecated after 10th April 2023.
+# The below mentioned IPs are deprecated since 10th April 2023. Newer RHEL images are already redirected to Public region for updates. If you have already added below IPs to your UDR/firewall, you are not required to remove these IPs until next update on this doc.
```

Comment:
Since we haven't retired the USGov RHUI yet or planned its deprecation, would it be accurate to say that below mentioned IPs are deprecated since 10th April, 2023?

Reply:
the date is already passed, but you are right let's not mention already deprecated here

Reply:
Still says that these IPs are deprecated

Reply:
Pushed commit now, thanks